resources/api-gateway-migrator/templates/job.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "api-gateway-migrator.name" . }}-job
+  namespace: {{ .Release.Namespace }}
+spec:
+  backoffLimit: 1
+  template:
+    metadata:
+      name: {{ template "api-gateway-migrator.name" . }}-job
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      serviceAccountName: {{ template "api-gateway-migrator.name" . }}-account
+      restartPolicy: Never
+      containers:
+        - name: migrator
+          image: eu.gcr.io/kyma-project/api-gateway-migrator:PR-7867
```

Comment:
Please set the image:tag as values

Reply:
The job is going to be a part of api-gateway chart. I'll configure it there.